<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="mainFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>988</width>
       <height>450</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>988</width>
       <height>450</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item alignment="Qt::AlignTop">
       <widget class="QFrame" name="frame_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 111, 166);
border-top-left-radius:5px;
border-top-right-radius:5px;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>THÔNG TIN PHIẾU NHẬP</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
border-bottom-left-radius:5px;
border-bottom-right-radius:5px;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>8</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_4">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="frame_6">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="font">
                  <font>
                   <pointsize>9</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Mã phiếu nhập</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="import_id_output">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Segoe UI</family>
                   <pointsize>-1</pointsize>
                   <weight>50</weight>
                   <italic>false</italic>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">#import_id_output{
	border: 2px solid rgb(226, 226, 226);
	border-radius:3px;
	font: 16px &quot;Segoe UI&quot;;
	background-color: rgb(247, 247, 247);
}
</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="font">
                  <font>
                   <pointsize>9</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Nhân viên nhập</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="employee_ouput">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Segoe UI</family>
                   <pointsize>-1</pointsize>
                   <weight>50</weight>
                   <italic>false</italic>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">#employee_ouput{
	border: 2px solid rgb(226, 226, 226);
	border-radius:3px;
	font: 16px &quot;Segoe UI&quot;;
	background-color: rgb(247, 247, 247);
}
</string>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_8">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="font">
                  <font>
                   <pointsize>9</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Thời gian tạo</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="time_create">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Segoe UI</family>
                   <pointsize>-1</pointsize>
                   <weight>50</weight>
                   <italic>false</italic>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">#time_create{
	border: 2px solid rgb(226, 226, 226);
	border-radius:3px;
	font: 16px &quot;Segoe UI&quot;;
	background-color: rgb(247, 247, 247);
}
</string>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="detailImportTable">
           <property name="styleSheet">
            <string notr="true">QTableWidget {
    font: 13px &quot;Segoe UI&quot;;
}

QHeaderView::section {
    background-color: #2980b9;
    color: white;
    padding: 6px;
	border: 1px solid rgb(191, 191, 191);
    font-weight: bold;
}

QTableWidget::item {
    padding: 6px;
    border: none;
}

QTableWidget::item:selected {
    background-color: #cce5ff;
	border: 1px solid #2980b9;
    color: black;

}


QTableWidget QScrollBar:vertical {
    background: #f0f0f0;
    width: 10px;
    margin: 0px 0px 0px 0px;
    border: none;
    border-radius: 5px;
}

QTableWidget QScrollBar::handle:vertical {
    background: #7aa5d2;
    min-height: 30px;
    border-radius: 5px;
}

QTableWidget QScrollBar::handle:vertical:hover {
    background: #5a92c4;
}

QTableWidget QScrollBar::add-line:vertical, 
QTableWidget QScrollBar::sub-line:vertical {
    background: none;
    height: 0px;
}

QTableWidget QScrollBar::add-page:vertical, 
QTableWidget QScrollBar::sub-page:vertical {
    background: none;
}
</string>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhNone</set>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QFrame" name="frame_15">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <property name="spacing">
             <number>32</number>
            </property>
            <item>
             <widget class="QToolButton" name="export_pdf_btn">
              <property name="minimumSize">
               <size>
                <width>140</width>
                <height>35</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#export_pdf_btn{
	background-color: rgb(0, 138, 202);
	color: rgb(255, 255, 255);
	font: 15px &quot;Segoe UI&quot;;
	border-radius:5px;
}
#export_pdf_btn::hover{	
	background-color: rgb(0, 85, 127);
}</string>
              </property>
              <property name="text">
               <string>Xuất file PDF</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="cancel_btn">
              <property name="minimumSize">
               <size>
                <width>140</width>
                <height>35</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#cancel_btn{
	background-color: rgb(255, 55, 55);
	color: rgb(255, 255, 255);
	font: 15px &quot;Segoe UI&quot;;
	border-radius:5px;
}

#cancel_btn::hover{
	
	background-color: rgb(203, 0, 0);
}</string>
              </property>
              <property name="text">
               <string>Hủy bỏ</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
